scrapi/processing/cassandra.py

```diff
@@ -32,7 +32,7 @@ def process_normalized(self, raw_doc, normalized):
             contributors=copy_to_unicode(json.dumps(normalized['contributors'])),
             description=copy_to_unicode(normalized.get('description')),
             uris=copy_to_unicode(json.dumps(normalized['uris'])),
-            providerUpdatedDateTime=parse(normalized['providerUpdatedDateTime']),
+            providerUpdatedDateTime=parse(normalized['providerUpdatedDateTime']).replace(tzinfo=None),
```

Comment:
Yeah, Cassandra thing. cqlengine loses timezone information when it saves datetimes, and since we are moving away from it I just don't think it's worth it.